jetty-documentation/src/main/asciidoc/administration/startup/startup-unix-service.adoc

```diff
@@ -72,30 +65,24 @@ Prepare some empty directories to work with.
 The directory purposes are as follows:
 
 /opt/jetty::
-  Where the Jetty Distribution will be unpacked into
+Where the Jetty Distribution will be unpacked into
 /opt/web/mybase::
-  Where your specific set of webapps will be located, including all of
-  the configuration required of the server to make them operational.
+Where your specific set of webapps will be located, including all of the configuration required of the server to make them operational.
 /opt/jetty/temp::
-  This is the temporary directory assigned to Java by the Service Layer
-  (this is what Java sees as the `java.io.tmpdir` System Property)
-  +
-  This is intentionally kept separate from the standard temp directory
-  of `/tmp`, as this location doubles as the Servlet Spec work
-  directory. (It is our experience that the standard temp directory is
-  often managed by various cleanup scripts that wreak havoc on a long
-  running Jetty server
-
-Jetty $\{project.version} requires Java 7 (or greater) to run. Make sure
-you have it installed.
+This is the temporary directory assigned to Java by the Service Layer (this is what Java sees as the `java.io.tmpdir` System Property).
++
+This is intentionally kept separate from the standard temp directory of `/tmp`, as this location doubles as the Servlet Spec work directory.
+It is our experience that the standard temp directory is often managed by various cleanup scripts that wreak havoc on a long running Jetty server.
+
+Jetty $\{project.version} requires Java 7 (or greater) to run.
```

Comment:
The $\{project.version} does not substitute for anything on the generated doc.

Also, jetty-9.3 and above requires Java 8 or above to run.